experimental-examples/tina-cloud-starter-iframe/tina/config.tsx

```diff
@@ -190,6 +192,35 @@ const config = defineStaticConfig({
                   { label: "Default", value: "default" },
                   { label: "Primary", value: "primary" },
                 ],
+                ui: {
+                  component: wrapFieldsWithMeta((props) => {
+                    return (
+                      /* @ts-ignore */
```

Comment:
This part is annoying, it's also used on the old version, anyone with a better solution?